build.gradle.kts

```diff
@@ -8,6 +8,8 @@ object Versions {
     const val flyway = "8.0.3"
     const val hikari = "5.0.0"
     const val jackson = "2.13.0"
+    const val kafka = "2.7.0"
+    const val kafkaEmbedded = "2.7.0"
```

Comment:
Kan vurdere nyere versjon 2.8.0 på disse to her.

Reply:
Prøver det